### PR TITLE
MISC/doc: 80limit + prettify generated documentation + tests for understanding

### PR DESCRIPTION
**Pull request by dannywillems**

### CODE REVIEW

File: poly-commitment/src/commitment.rs [hunk 1]
```diff
@@ -456,10 +460,17 @@ pub struct Evaluation<G>
 where
     G: AffineCurve,
 {
-    /// The commitment of the polynomial being evaluated
+    /// The commitment of the polynomial being evaluated.
+    /// Note that PolyComm contains a vector of commitments, which handles the
```
Comment (querolita) [inline poly-commitment/src/commitment.rs:464]: Thanks for this, it is the thing I always forget about and always keep asking myself why `[0]`...

File: poly-commitment/src/commitment.rs [hunk 2]
```diff
@@ -483,14 +498,31 @@ where
...
 
+/// This function populates the parameters `scalars` and `points`.
+/// It iterates over the evaluations and adds each commitment to the
+/// vector `points`.
+/// The parameter `scalars` is populated with the values:
+/// `rand_base * polyscale^i` for each commitment.
+/// For instance, if we have 3 commitments, the `scalars` vector will
+/// contain the values
+/// ```text
+/// [rand_base, rand_base * polyscale, rand_base * polyscale^2]`
```
Comment (querolita) [inline poly-commitment/src/commitment.rs:509]: nice to have some examples in the docs

File: poly-commitment/src/commitment.rs [hunk 3]
```diff
@@ -502,11 +534,35 @@ pub fn combine_commitments<G: CommitmentCurve>(
...
         }
     }
 }
 
+/// Combine the (chunked) evaluations of multiple polynomials.
+/// This function returns the accumulation of the evaluations, scaled by
+/// `polyscale`.
+/// If no evaluation is given, the function returns an empty vector.
+/// It does also suppose that for each evaluation, the number of evaluations is
+/// the same. It is not constrained yet in the interface, but it should be. If
```
Comment (querolita) [inline poly-commitment/src/commitment.rs:548]: Awesome catch

File: poly-commitment/src/commitment.rs [hunk 4]
```diff
@@ -526,9 +582,10 @@ pub fn combine_evaluations<G: CommitmentCurve>(
         .filter(|x| !x.commitment.elems.is_empty())
     {
         // iterating over the polynomial segments
-        for j in 0..evaluations[0].len() {
-            for i in 0..evaluations.len() {
-                acc[i] += evaluations[i][j] * xi_i;
+        for chunk_idx in 0..evaluations[0].len() {
```
Comment (querolita) [inline poly-commitment/src/commitment.rs:585]: I like the meaningful iterator names

File: poly-commitment/src/commitment.rs [hunk 5]
```diff
@@ -674,16 +734,8 @@ where
 }
 
 impl<G: CommitmentCurve> SRS<G> {
-    /// This function verifies batch of batched polynomial commitment opening proofs
-    ///     batch: batch of batched polynomial commitment opening proofs
-    ///          vector of evaluation points
```
Comment (querolita) [inline poly-commitment/src/commitment.rs:679]: so you're deleting this because now one can have info about the input types with the new docs?

**Review by querolita — APPROVED** [review]

**Review by volhovm — APPROVED** [review]

File: poly-commitment/src/commitment.rs (hunk 5 above)
Comment (dannywillems) [reply poly-commitment/src/commitment.rs:679]: I removed because the comment seems outdated.

**Pull request merged**